lib/vsc/utils/rest.py

```diff
@@ -104,7 +104,7 @@ def __init__(self, url, username=None, password=None, token=None, token_type='To
         elif token is not None:
             self.auth_header = '%s %s' % (token_type, token)
 
-    def get(self, url, headers={}, **params):
+    def get(self, url, headers=None, **params):
```

Comment:
these all pass this arg to self.request, and I've done that in self.request, so its ok.